### DISCUSSION TITLE
Yomi Hunter accused Magick of carrying a virus

### DISCUSSION THREAD

**Discussion by RonaldBolado** [ImageMagick/ImageMagick#2668] [General]
ImageMagick-7.0.10-31-Q16-HDRI-x64-dll.exe

https://www.virustotal.com/gui/file/e8248690f23170fc3019642402a234e6aa62aa06e399b3c8decf32ba97a09423/detection

What is up with that?

**Comment by dlemstra** [comment]
That is a false possitive.
